NewSpring/corinth
# chore: Update Campaign Card

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR updates the BrandedCard to remove the play icon and show a label of "Sermon Notes" when on the campaign item we're live.

Not Live:
![Screenshot 2020-01-27 14 53 16](https://user-images.githubusercontent.com/3229463/73216784-a4ef5d80-4124-11ea-8b7f-c5dcb89901e4.png)
Live:
![Screenshot 2020-01-28 14 54 27](https://user-images.githubusercontent.com/3229463/73301176-4982a580-41e0-11ea-92c8-d8ead2c120c1.png)


### How do I test this PR?
Run the app locally and fire up that simulator. You will probably have to edit the code in `BrandedCard.js` to simulate that `isLive` is true (unless you're testing when we're _actually_ live). You should see the difference shown in the screenshots above.

You'll want to make sure that the other items in the feed show the cards and pills correctly. There should be no change to them. This change should only affect the campaign item card.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
